pkg/types/validation/installconfig_test.go

```diff
@@ -1999,6 +1999,19 @@ func TestValidateInstallConfig(t *testing.T) {
 			}(),
 			expectedError: "platform.vsphere.apiVIPs: Required value: must specify VIP for API, when VIP for ingress is set",
 		},
+		{
+			name: "GCP Create Firewall Rules should return error if used WITHOUT tech preview when not enabled",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Platform = types.Platform{
+					GCP: validGCPPlatform(),
+				}
+				c.GCP.CreateFirewallRules = gcp.CreateFirewallRulesDisabled
+
+				return c
+			}(),
+			expectedError: "platform.gcp.createFirewallRules: Forbidden: the TechPreviewNoUpgrade feature set must be enabled to use this field",
+		},
```

Comment:
Do we need a test case here verify that this feature is not blocked when  c.FeatureSet is set to "TechPreviewNoUpgrade" ?